### ISSUE TITLE
[MODULE] - VADER sentiment (scores only) 

### ISSUE THREAD

**Issue by LeonardPuettmannKern** [code-kern-ai/bricks#276]
**Please describe the module you would like to add to bricks**
The VADER sentiment brick will be split into a module only for classification and a generator that returns the scores. The classification one keeps the previous issue ID. This issue is meant to the VADER sentiment module that returns the scores. Please post everything regarding the VADER sentiment score brick here.

**Do you already have an implementation?**
```python
from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
import json

ATTRIBUTE: str = "text" # only text attributes

def vader_sentiment(record):
    analyzer = SentimentIntensityAnalyzer()
    text = record[ATTRIBUTE].text

    vs = analyzer.polarity_scores(text)
    return json.dumps(vs)
```

**Additional context**
The issue for the VADER sentiment classifier can be found here -> #157

**Comment by LeonardPuettmannKern** [comment]
Part of the bricks restructure. This issue will be closed once the restructure is done.

**Comment by LeonardPuettmannKern** [comment]
Implemented with #283
